components/SingleArticleContainer.jsx

```diff
@@ -0,0 +1,22 @@
+export const SingleArticleContainer = ({ selectedArticle, isLoading }) => {
+  console.log(selectedArticle);
```

Comment:
Make sure to remove console logs before committing/pushing.

Reply:
There's always one that gets away!